### PR TITLE
disable qr code when set disableExport to true

### PR DESCRIPTION
**Pull request by Sneezry**
https://github.com/Authenticator-Extension/Authenticator/issues/461

### CODE REVIEW

**Review by mymindstorm — CHANGES_REQUESTED** [review]
Instead of using `store/Entry.ts`, use the variable `exportDisabled` in `store/Menu.ts`

**Comment by Sneezry** [comment]
Done.

File: src/store/Menu.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export class Menu implements Module {
         exportDisabled: await ManagedStorage.get("disableExport"),
         enforcePassword: await ManagedStorage.get("enforcePassword"),
         enforceAutolock: await ManagedStorage.get("enforceAutolock"),
+        showQrDisabled: await ManagedStorage.get("disableExport"),
```
Comment (mymindstorm) [inline src/store/Menu.ts]: Why did you add a line here? The exact same thing happens on line 13

Comment (Sneezry) [reply src/store/Menu.ts]: Ooooops

**Review by mymindstorm — APPROVED** [review]
